input/pagecontent/guidance.md

```diff
@@ -1,16 +1,15 @@
+The Data Exchange for Quality Measure (DEQM) Implementation Guide defines the interactions for data exchange scenarios in the Quality Measure Ecosystem.
+
+The first interaction is when a Producer, such as a practitioner, or owner of data needs to exchange that data with a Consumer of that data, such as a payer, a registry or public health. We call this the Data Exchange Scenario. Examples of this interaction might be when a provider has patient information from a recent visit that he needs to share with a payer under a value based contract. There may also be use cases where the Producer in this scenario is a Payer and needs to exchange data with a Provider.
 
-The second scenario defined in this guide is when a Reporter needs to exchange a measure report with a Receiver.  This guide addresses the Individual Measure Reporting and the Summary Reporting.  As an example, Individual Measure Reports may be used by hospitals acting as the Reporter to report a specific measure to a payer acting as a Receiver.  Similarly, Summary Measure Reports may be used to report yearly eCQM results on a specific measure.
+The second scenario defined in this guide is when a Reporter needs to exchange a measure report with a Receiver. This guide addresses the Individual Measure Reporting and the Summary Reporting. As an example, Individual Measure Reports may be used by hospitals acting as the Reporter to report a specific measure to a payer acting as a Receiver. Similarly, Summary Measure Reports may be used to report yearly eCQM results on a specific measure.
 
-This version of the guide adds the Gaps in Care Reporting Scenario. The Gaps in Care Reporting is used to report the [open and/or closed gaps] for quality measures over a [gaps through period] specified by a Client. Optionally, it is also used to report details to the open gaps identified and mitigation steps for addressing them. It further provides capability of associating clinical data included in the report with the population criteria (i.e. denominator, numerator) of a measure that they apply to.
+The third scenario defined in this guide is the Gaps in Care Reporting. The Gaps in Care Reporting is used to report the open and/or closed gaps for quality measures over a gaps through period specified by a Client. Optionally, it is also used to report details to the open gaps identified and mitigation steps for addressing them. It further provides capability of associating clinical data included in the report with the population criteria (i.e. denominator, numerator) of a measure that they apply to.
```

Comment:
1) The first sentence here differs from what's in the Resolution. 

"The third interaction is Gaps in Care Reporting. Gaps in Care Reporting is used to report the open and/or closed gaps for quality measures over a gaps through period specified by a Client."

2) Do not remove the hyperlink from "open and/or closed gaps", "gaps through period"